realm/src/main/java/io/realm/base/BaseRealm.java

```diff
@@ -0,0 +1,501 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.realm.base;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.os.Message;
+
+import java.io.Closeable;
+import java.io.File;
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import io.realm.BuildConfig;
+import io.realm.RealmChangeListener;
+import io.realm.RealmConfiguration;
+import io.realm.internal.RealmProxyMediator;
+import io.realm.internal.SharedGroup;
+import io.realm.internal.SharedGroupManager;
+import io.realm.internal.Table;
+import io.realm.internal.android.DebugAndroidLogger;
+import io.realm.internal.android.ReleaseAndroidLogger;
+import io.realm.internal.log.RealmLog;
+
+/**
+ * Base class for all Realm instances.
+ * Specialized Realms do not not share the same underlying native resources.
```

Comment:
"not not" -> "not"; "the same" -> ""

Don't you mean that two instances don't share native resources (except the file)?